Module3/SQL_ASSIGNMENT_SUBMISSION.txt

```diff
@@ -0,0 +1,2320 @@
+1.List all the columns of the Salespeople table. 
+
+mysql> DESC SALESPEOPLE;
++-------+--------------+------+-----+---------+-------+
+| Field | Type         | Null | Key | Default | Extra |
++-------+--------------+------+-----+---------+-------+
+| SNUM  | int(11)      | YES  |     | NULL    |       |
+| SNAME | varchar(100) | YES  |     | NULL    |       |
+| CITY  | varchar(100) | YES  |     | NULL    |       |
+| COMM  | int(11)      | YES  |     | NULL    |       |
++-------+--------------+------+-----+---------+-------+
+4 rows in set (0.02 sec)
+
+
+OR
+
+
+mysql> SELECT COLUMN_NAME 
+FROM INFORMATION_SCHEMA.COLUMNS 
+WHERE TABLE_NAME = 'SALESPEOPLE';
+
++-------------+
+| COLUMN_NAME |
++-------------+
+| SNUM        |
+| SNAME       |
+| CITY        |
+| COMM        |
++-------------+
+4 rows in set (0.02 sec)
+
+
+OR
+
+mysql> SHOW COLUMNS 
+FROM SALESPEOPLE;
++-------+--------------+------+-----+---------+-------+
+| Field | Type         | Null | Key | Default | Extra |
++-------+--------------+------+-----+---------+-------+
+| SNUM  | int(11)      | YES  |     | NULL    |       |
+| SNAME | varchar(100) | YES  |     | NULL    |       |
+| CITY  | varchar(100) | YES  |     | NULL    |       |
+| COMM  | int(11)      | YES  |     | NULL    |       |
++-------+--------------+------+-----+---------+-------+
+4 rows in set (0.02 sec)
+
+2. List all customers with a rating of 100. 
+mysql> SELECT * 
```

Comment:
Use specific column names instead of asterisks.

Reply:
Instead of using the asterisk symbol (*) in all SQL queries, use the proper column name.